### PR TITLE
Fixed the bug

### PR DESCRIPTION
**Pull request by mahdishahsavar**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -12,9 +12,10 @@
 from steganography.user_interface import Steganography
 
 # Fonts
+pygame.init()
 font = pygame.font.Font(None, 36)
 
-PROBLEMS = [PasswordCracker(), NetworkSniffer(), Steganography()]
+PROBLEMS = [PasswordCracker(), NetworkSniffer()]#, Steganography()]
```
Comment (PeterLenon) [inline main.py:18]: Is there any reason you omitted the steganography challenge?

File: main.py [hunk 2]
```diff
@@ -45,7 +45,10 @@ def ask_question_with_node_class(node):
     for problem in PROBLEMS:
         if problem.id == challenge_id:
             challenge = problem
-    challenge.run()
+    if challenge is not None:
+        challenge.run()
+    else:
+        print("No valid challenge found for ID:", challenge_id)
```
Comment (PeterLenon) [inline main.py:51]: great validation check here!

**Review by PeterLenon — CHANGES_REQUESTED** [review]
This PR seems good to me but please do address the comment I left about ommitting the last challenge I placed. Thanks

**Review by PeterLenon — APPROVED** [review]

File: main.py (hunk 1 above)
Comment (mahdishahsavar) [reply main.py:18]: It is not working correctly on my side. I didn't have time to check exactly what the problem is, but I'll look into it tonight.

**Pull request merged**